OpenRA.Mods.Common/Widgets/LabelWithTooltipWidget.cs

```diff
@@ -1,4 +1,4 @@
-#region Copyright & License Information
+﻿#region Copyright & License Information
```

Comment:
As a rule of thumb. The whole OpenRA.Mods.Common is already StyleCop clean so please don't commit any changes to that directory.